src/MangaJet/app/src/main/res/layout/activity_extra_settings.xml

```diff
@@ -0,0 +1,114 @@
+
+        <TextView
+            android:id="@+id/searchMaxResultsText"
+            android:layout_width="0dp"
+            android:layout_gravity="center"
+            android:layout_height="wrap_content"
+            android:layout_weight="70"
+            android:textSize="16sp"
+            android:text="@string/extra_maxSearchResInfo" />
+
+        <EditText
+            android:id="@+id/searchMaxResultsValue"
+            android:layout_width="0dp"
+            android:layout_gravity="center"
+            android:layout_height="wrap_content"
+            android:layout_weight="30"
+            android:minEms="3"
+            android:inputType="numberDecimal" />
+    </LinearLayout>
+
+    <LinearLayout
+        android:layout_marginStart="@dimen/inner_horizontal_margin"
+        android:layout_marginEnd="@dimen/inner_horizontal_margin"
+        android:layout_marginTop="@dimen/inner_vertical_margin"
+        android:id="@+id/forYouMaxTags"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:orientation="horizontal"
+        android:weightSum="100"
+        app:layout_constraintTop_toBottomOf="@id/searchMaxResults"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintEnd_toEndOf="parent">
+
+        <TextView
+            android:id="@+id/forYouMaxTagsText"
+            android:layout_width="0dp"
+            android:layout_gravity="center"
+            android:layout_height="wrap_content"
+            android:layout_weight="70"
+            android:textSize="16sp"
+            android:text="@string/extra_maxForYouTagsInfo" />
+
+        <EditText
+            android:id="@+id/forYouMaxTagsValue"
+            android:layout_width="0dp"
+            android:layout_gravity="center"
+            android:layout_height="wrap_content"
+            android:layout_weight="30"
+            android:minEms="3"
+            android:inputType="numberDecimal" />
+    </LinearLayout>
+
+    <LinearLayout
+        android:layout_marginStart="@dimen/inner_horizontal_margin"
+        android:layout_marginEnd="@dimen/inner_horizontal_margin"
+        android:layout_marginTop="@dimen/inner_vertical_margin"
+        android:id="@+id/forYouMaxResults"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:orientation="horizontal"
+        android:weightSum="100"
+        app:layout_constraintTop_toBottomOf="@id/forYouMaxTags"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintEnd_toEndOf="parent">
+
+        <TextView
+            android:id="@+id/forYouMaxResultsText"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_weight="70"
+            android:textSize="16sp"
+            android:layout_gravity="center"
+            android:text="@string/extra_maxForYouMangasInfo" />
+
+        <EditText
```

Comment:
Can we use slider with textView instead of EditText? This way boundaries will be more Obvious to user.
And we can write in a small textbox amount in numbers. Like this
![image](https://user-images.githubusercontent.com/19706253/169138457-a4e7dc16-dcbd-42e0-ad28-9db886099a58.png)